### PR TITLE
Upgrade molo to v4

### PR DESCRIPTION
**Pull request by nathanbegbie**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@saeedmb8 please review

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,9 @@
-molo.core==3.12.3
-wagtailmedia==0.1.1
+molo.core==4.0.0
```
Comment (smarzban) [inline requirements.txt:1]: @nathanbegbie please check if there was anything added from 3.12.3 to 4.0.0 in molo core that we need to add to free basics. same goes for the plugins

**Comment by nathanbegbie** [comment]
@saeedmb8 please review again

**Comment by smarzban** [comment]
👍

**Pull request merged**